src/webui/www/private/confirmdeletion.html

```diff
@@ -3,7 +3,7 @@
 
 <head>
     <meta charset="UTF-8" />
-    <title>QBT_TR(Deletion confirmation - qBittorrent)QBT_TR[CONTEXT=confirmDeletionDlg]</title>
+    <title>QBT_TR(Remove torrents(s))QBT_TR[CONTEXT=confirmDeletionDlg]</title>
```

Comment:
I think you have a typo here and in the other webui files.
Shouldn't it be `Remove torrent(s)`? (one `s`)

Reply:
Sorry about that, fixed.